apps/crn-frontend/src/network/teams/Outputs.tsx

```diff
@@ -133,8 +132,7 @@ const Outputs: React.FC<OutputsProps> = ({ teamId }) => {
           teamId={teamId}
           searchQuery={debouncedSearchQuery}
           filters={filters}
-          hasOutputs={hasOutputs}
-          ownTeam={ownTeam}
+          userAssociationMember={userAssociationMember}
           contactEmail={team?.pointOfContact?.email}
```

Comment:
This is also funky, if there isn't a team we should display 404.

Reply:
Not for you to fix just an observation

Reply:
I am going to send the team not the team id, as the parent component also retrieves a team by id and renders or not all the child component based on that team (which can be null and for that case it does render the NotFoundPage )